docs/products/clickhouse/howto/manage-databases-tables.md

```diff
@@ -33,20 +33,68 @@ can create a database through the web interface of the
     necessary customizations and run secondary queries to grant access
     to the admin user.
 
+</TabItem>
+<TabItem value="cli" label="SQL">
+
+**Limitations**
+
+- Only `avnadmin` users can create databases in SQL.
+- You can create a database in SQL with the `Replicated` database engine only.
+
+To create a database in SQL, run the following SQL command:
+
+```sql
+CREATE DATABASE DATABASE_NAME
+ENGINE = Replicated
+```
+
+For example:
+
+```sql
+CREATE DATABASE transactions
+ENGINE = Replicated;
+```
+
+</TabItem>
+</Tabs>
+
 ## Delete a database
 
+:::important
+Deleting a database is irreversible and permanently removes the database along
+with all its tables and data.
+:::
+
+You can delete a database either in the [Aiven Console](https://console.aiven.io/) or
+using an SQL client (for example, the
+[ClickHouse client](/docs/products/clickhouse/howto/connect-with-clickhouse-cli)):
+
+<Tabs groupId="group1">
+<TabItem value="console" label="Aiven Console" default>
 1.  Log in to the [Aiven Console](https://console.aiven.io/), and select
     your service from the <ConsoleLabel name="Services"/> page.
 1. In the sidebar, click <ConsoleLabel name="databasesandtables"/>.
 1. In the **Databases and tables** list, find your database and click
     <ConsoleLabel name="actions"/> > <ConsoleLabel name="deletedatabase"/>.
+</TabItem>
+<TabItem value="cli" label="SQL">
 
-:::note
-If you try adding or removing a database in for your Aiven for
-ClickHouse service through the command line, you'll encounter an
-exception `Not enough privileges.(ACCESS_DENIED)`. Use the Aiven
-web interface to add or remove a database.
-:::
+**Limitation**: Only `avnadmin` users can delete databases in SQL.
```

Comment:
1. users->user.
2. Actually it is a bit more complicated than that. By default only `aiven` user can delete a database, but `aiven` user can grant deleting a database to another user if he wants.